From: "Chris M. Simon" <cmsimon@lavabit.com>
Date: Tue, 23 Nov 2010 04:29:14 +0000
Subject: Added Cities.

---
 na.phone | 5 +++++
 1 file changed, 5 insertions(+)

diff --git a/na.phone b/na.phone
index ff41dc7..9cc657c 100644
--- a/na.phone
+++ b/na.phone
@@ -922,6 +922,7 @@
 502:Louisville:Kentucky:KY
 503:Aloha:Oregon:OR
 503:Beaverton:Oregon:OR
+503:Fairview:Oregon:OR
 503:Gresham:Oregon:OR
 503:Hillsboro:Oregon:OR
 503:Keizer:Oregon:OR
@@ -930,10 +931,14 @@
 503:Milwaukie:Oregon:OR
 503:Oregon City:Oregon:OR
 503:Portland:Oregon:OR
+503:Rock Away:Oregon:OR
 503:Salem:Oregon:OR
 503:Tigard:Oregon:OR
+503:Troutdale:Oregon:OR
 503:Tualatin:Oregon:OR
 503:West Linn:Oregon:OR
+503:Wilsonville:Oregon:OR
+503:Wood Village:Oregon:OR
 503:Woodburn:Oregon:OR
 504:Chalmette:Louisiana:LA
 504:Harvey:Louisiana:LA
